tools/lib/test_server.py

```diff
@@ -0,0 +1,86 @@
+from __future__ import print_function
+
+import os
+import subprocess
+import sys
+import time
+
+from contextlib import contextmanager
+from typing import (Any, Iterator)
+
+try:
+    import django
+    import requests
```

Comment:
Why did you split this from the `from typing import Iterator` block in run-casper?  

It seems like this is the wrong divide between the pieces, because this script will always fail when outside the venv on Python 2 (because `typing` is only available in the venv).